Spelling in github actions broken?

The spell checker seems to be broken.  You can see it failing in https://github.com/intel/cve-bin-tool/pull/1467 and https://github.com/intel/cve-bin-tool/pull/1469

I did a little bit of experimenting and it looks like it might be related to the github api when it tries to submit comments. 

```
Checking 41 files
(...Spell check...)
Warning: doc/MANUAL.md: line 3, columns 1-8, Warning - `typtoto` is not a recognized word. (unrecognized-spelling)
(...Compare expect with new output...)
(...New output...)
(...Unrecognized...)
Preparing a comment for pull_request
https://api.github.com/repos/intel/cve-bin-tool/issues/1469/comments
curl: (6) Could not resolve host: null
Error: Process completed with exit code 6.
```

I thought maybe getting the latest .yml from the spelling checker folk would address the issue (that's what's in #1469 ) but that doesn't seem to have resolved the problem.  If anyone's got any insights, I'm currently drawing a blank on what to try next.  Pinging @Molkree and @P0intMaN who might be most likely to know how to fix this.

--- a/.github/workflows/spelling.yml
+++ b/.github/workflows/spelling.yml
@@ -1,5 +1,5 @@
 name: Spell checking
-on: [push, pull_request_target]
+on: [push, pull_request]
 
 jobs:
   spelling:
@@ -8,3 +8,5 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: check-spelling/check-spelling@v0.0.19
+      with:
+        post_comment: '0'

